.github/CODEOWNERS

```diff
@@ -231,6 +231,7 @@
 /core/templates/services/exploration-html-formatter.service*.ts @kevintab95
 /core/templates/services/image-upload-helper.service*.ts @kevintab95
 /core/templates/services/validators.service*.ts @kevintab95
+/core/domain/blog*.py @iamprayush
```

Comment:
This file is not related to Exploration, right? Maybe create a separate section `Blog post project` and add all the files there* (The project sections are placed in alphabetical sorted order*)

Reply:
Done.